src/transformers/field-relation.transform.spec.ts

```diff
@@ -0,0 +1,66 @@
+import type { CmsFieldBase, CmsFieldRelation } from 'decap-cms-core';
+import z from 'zod';
+
+import { transformRelationField } from './field-relation.transform.js';
+
+describe('field-relation.transform', () => {
+  // Constructs a base relation field configuration with overrides
+  function buildField(
+    partial: Partial<CmsFieldBase & CmsFieldRelation>,
+  ): CmsFieldBase & CmsFieldRelation {
+    return {
+      name: 'test_field',
+      widget: 'relation',
+      collection: 'authors',
+      multiple: false,
+      required: true,
+      search_fields: ['name'],
+      value_field: 'name',
+      display_fields: ['name'],
+      ...partial,
+    };
+  }
+
+  // Normalises quotes in the cptime output for consistent comparison
+  function normaliseCptime(cptime: string): string {
```

Comment:
This was only added because of a battle between eslint and prettier, so it's there to satisfy the IDE only.

I couldn't get eslint and prettier rules to agree on quotes.

Reply:
Ah, I see. But badly configured tooling shouldn't force us to do such things. I'll align the configuration.